frontend/projects/upgrade/src/app/features/dashboard/home/root/home.component.html

```diff
@@ -2,7 +2,7 @@
   <h1 class="title ft-32-700">{{ 'global.experiment.title' | translate }}</h1>
   <span class="subtitle ft-16-400" [innerHTML]="'home.experiment.text.subtitle' | translate"></span>
 
-  <ng-container *ngIf="isLoadingExperiments$ | async; else loadingExperimentState">
```

Comment:
The problem with using this observable here is that, rather than only re-generating the 'experiment-list' component once, on initial load, it does so every time the experiment list changes. One effect this has is that we lose the ability to enter more than one character at a time in the search field, which is probably not what we want.